querier_commands_test.go

```diff
@@ -73,7 +73,8 @@ func (s *ReformSuite) TestInsertWithPrimaryKey() {
 }
 
 func (s *ReformSuite) TestInsertWithStringPrimaryKey() {
-	project := &Project{ID: "new", End: pointer.ToTime(time.Now().Truncate(24 * time.Hour))}
+	today := time.Now().Truncate(24 * time.Hour)
```

Comment:
I'm not sure if it's the case here, but in the recent Radio-T they discussed a possible problem with such rounding on daylight saving boundaries. Some additional info: http://stackoverflow.com/questions/25254443/